odoo/addons/openupgrade_records/lib/apriori.py

```diff
@@ -35,6 +35,8 @@
     # OCA/stock-logistics-workflow:
     'product_customer_code_picking':
         'product_supplierinfo_for_customer_picking',
+    # OCA/website
+    'website_img_bg_style': 'website',
```

Comment:
wrong, this should go below, in the `merged_modules` section

Reply:
Please fix this

Reply:
I have changed this myself.